jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/MemoryResource.java

```diff
@@ -37,7 +35,7 @@ public class MemoryResource extends Resource
 {
     private final URI _uri;
     private final Instant _created = Instant.now();
-    private final byte[] _bytes;
+    private final byte[] _bytes; // TODO a ByteBuffer would be nicer
```

Comment:
Can that be done?

Reply:
That's actually not needed. I've removed that comment.